cgroup_top.py

```diff
@@ -307,6 +337,18 @@ def collect(measures):
         for cgroup in cgroups(CGROUP_MOUNTPOINTS['memory']):
             cur[cgroup.name]['memory.usage_in_bytes'] = cgroup['memory.usage_in_bytes']
             cur[cgroup.name]['memory.limit_in_bytes'] = min(int(cgroup['memory.limit_in_bytes']), measures['global']['total_memory'])
+    elif HAS_OPENVZ:
+        user_beancounters = get_user_beacounts('root')
+        for line in user_beancounters:
```

Comment:
Field `privvmpages.l' is not available in JSON output - and we cannot get limit.